setup.cfg

```diff
@@ -28,6 +28,9 @@ install_requires =
     toolchest
 
 [options.extras_require]
+celery =
+    celery
```

Comment:
(praise) I really like how self-contained the additions are, here and in the Pipfile, install instructions, etc. It works really neatly.